<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoutesViewWindow</class>
 <widget class="QDialog" name="RoutesViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Просмотр маршрутов</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/new/prefix1/icons1/samolet.jpg</normaloff>:/new/prefix1/icons1/samolet.jpg</iconset>
  </property>
  <widget class="QTableView" name="routesView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>73</y>
     <width>512</width>
     <height>189</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>268</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>54</y>
     <width>289</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>                                                                  Доступные рейсы</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depFilter">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>499</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пункт отправления                                                                                                      Пункт назначения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="desFilter">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RoutesViewWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>depFilter</sender>
   <signal>editingFinished()</signal>
   <receiver>RoutesViewWindow</receiver>
   <slot>startFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desFilter</sender>
   <signal>editingFinished()</signal>
   <receiver>RoutesViewWindow</receiver>
   <slot>startFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startFilter()</slot>
 </slots>
</ui>
